cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/EndPointField.java

```diff
@@ -69,4 +70,19 @@ public String toString() {
       ", field='" + field + '\'' +
       '}';
   }
+
+  /**
+   * Checks for validity of an EndPointField.
+   *
+   * If wrangler is used in a pipeline to drop a column, the source EndPointField corresponding to
```

Comment:
We shouldn't take about wrangler in the platform code.

Reply:
+1, just mention in which scenario the endpoint will be null and invalid